.azure-pipelines/docker-sonic-vs/Dockerfile

```diff
@@ -4,4 +4,8 @@ ARG docker_container_name
 
 ADD ["wheels", "/wheels"]
 
-RUN pip3 install --no-deps --force-reinstall /wheels/sonic_utilities-1.2-py3-none-any.whl
+# Uninstalls only sonic-utilities and does not impact its dependencies
```

Comment:
`docker-sonic-vs` is retrieved, then the new sonic-utilities package is installed with most recent modifications. Problem happens if the new package has new external libs or one of the libs is updated as using the command `pip3 install` with `--deps` just leave the deps unchanged.

I tried different ideas:
`pip3 install /wheels/sonic_utilities-1.2-py3-none-any.whl`
does not do anything because the package is already installed in the `docker-sonic-vs`

`pip3 install --force-reinstall /wheels/sonic_utilities-1.2-py3-none-any.whl`
does not work because it also forces the deps to be updated but some of the deps such as `sonic-py-common` are not available in public pip repo and are installed with the sonic image in `docker-sonic-vs`

`pip3 install --update /wheels/sonic_utilities-1.2-py3-none-any.whl`
does not update sonic-utilities package and only updates its dependencies

The proposed soln achieved the following:
- Force install sonic-utilities package
- Adding missing dependencies
- Upgrading out-dated dependencies
- Leaving dependencies provided by the sonic-build process unchanged.